### ISSUE TITLE
Unable to download php56 due to SSL cert problem

### ISSUE THREAD

**Issue by skithund** [Homebrew/homebrew-php#1784]
```
==> Upgrading homebrew/php/php56
==> Downloading https://www.php.net/get/php-5.6.10.tar.bz2/from/this/mirror

curl: (60) SSL certificate problem: Invalid certificate chain
More details here: http://curl.haxx.se/docs/sslcerts.html
```

Checking the URL with "curl -vv -k"
```
* Connected to www.php.net (84.234.75.250) port 443 (#0)
* TLS 1.2 connection using TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
* Server certificate: php.planeetta.net
```

My guess is that www.php.net is using geodns or similar to direct traffic to closest mirror, since the certificate is for php.planeetta.net, which is fi.php.net mirror.

Remove www from URL or change https to http.

**Comment by lifepillar** [comment]
Same here. Worked around the problem by removing `www.` in the value of `PHP_SRC_TARBALL` in `abstract-php-version.rb`, as suggested above.

**Comment by DsXack** [comment]
duplicate #1783

**Comment by alanthing** [comment]
Resolved in #1786.
